### PR TITLE
Code style cleanups

### PR DESCRIPTION
**Pull request by MarcinOrlowski**
Just pure code style cleanup (`final var`/`var`, var names, some methods renamed.

### CODE REVIEW

File: src/main/java/com/cburch/logisim/fpga/designrulecheck/Netlist.java [hunk 1]
```diff
@@ -2892,36 +2890,36 @@ private void WarningForGatedClock(
...
+                      SimpleDRCContainer.LEVEL_SEVERE,
+                      SimpleDRCContainer.MARK_INSTANCE | SimpleDRCContainer.MARK_WIRE);
           for (NetlistComponent comp : Components.get(i))
-            warn.AddMarkComponent(comp.GetComponent());
-          warn.AddMarkComponents(Wires.get(i));
+            warn.addMarkComponent(comp.getComponent());
+          warn.addMarkComponents(Wires.get(i));
           Reporter.Report.AddWarning(warn);
         }
         WarnedComponents.addAll(Components.get(i));
```
Comment (BFH-ktt1) [inline src/main/java/com/cburch/logisim/fpga/designrulecheck/Netlist.java:2925]: Maybe we should make this file smaller (I know that it is my code), ass 2927 lines is a lot :-)

**Review by BFH-ktt1 — APPROVED** [review]
I just left some comments that we may or may not implement

File: src/main/java/com/cburch/logisim/fpga/designrulecheck/ConnectionEnd.java [hunk 2]
```diff
@@ -37,37 +37,37 @@
...
     return myConnections.get(bitIndex);
   }
 
-  public boolean IsOutputEnd() {
+  public boolean isOutputEnd() {
     return isOutput;
   }
 
-  public int NrOfBits() {
+  public int nrOfBits() {
```
[the remark this answers is not in the document]
Comment (MarcinOrlowski) [reply src/main/java/com/cburch/logisim/fpga/designrulecheck/ConnectionEnd.java:58]: Done.

File: src/main/java/com/cburch/logisim/fpga/designrulecheck/Net.java [hunk 3]
```diff
@@ -80,33 +80,33 @@ public boolean AddParentBit(byte bitId) {
...
+    sourceNetsList.get(bitIndex).add(sourceNet);
     return true;
   }
 
   public void addTunnel(String tunnelName) {
     tunnelNames.add(tunnelName);
   }
 
-  public int BitWidth() {
+  public int bitWidth() {
```
[the remark this answers is not in the document]
Comment (MarcinOrlowski) [reply src/main/java/com/cburch/logisim/fpga/designrulecheck/Net.java:109]: Done

File: src/main/java/com/cburch/logisim/fpga/designrulecheck/Netlist.java [hunk 4]
```diff
@@ -180,241 +179,241 @@ public Net getNet() {
...
+    for (Component comp : myCircuit.getNonWires()) {
       String ComponentName = comp.getFactory().getHDLName(comp.getAttributeSet());
       if (!CompName.contains(ComponentName)) {
         CompName.add(ComponentName);
       }
     }
     drc.add(
-        new simpleDRCContainer(
-                MyCircuit,
+        new SimpleDRCContainer(
```
[the remark this answers is not in the document]
Comment (MarcinOrlowski) [reply src/main/java/com/cburch/logisim/fpga/designrulecheck/Netlist.java:374]: That's correct. But I do not understand what you talk about as for me the patch is correct, class name should be capitalized, but `MyCircuit` variable not.  Red is gone, green is added :)

File: src/main/java/com/cburch/logisim/fpga/designrulecheck/Netlist.java [hunk 5]
```diff
@@ -426,299 +425,298 @@ public int DesignRuleCheckResult(boolean IsTopLevel, ArrayList<String> Sheetname
...
           }
           SubcircuitFactory sub = (SubcircuitFactory) comp.getFactory();
-          LocalNrOfInportBubles =
-              LocalNrOfInportBubles + sub.getSubcircuit().getNetList().NumberOfInputBubbles();
-          LocalNrOfOutportBubles =
-              LocalNrOfOutportBubles + sub.getSubcircuit().getNetList().NumberOfOutputBubbles();
-          LocalNrOfInOutBubles =
-              LocalNrOfInOutBubles + sub.getSubcircuit().getNetList().NumberOfInOutBubbles();
+          localNrOfInportBubles =
+              localNrOfInportBubles + sub.getSubcircuit().getNetList().NumberOfInputBubbles();
```
[the remark this answers is not in the document]
Comment (MarcinOrlowski) [reply src/main/java/com/cburch/logisim/fpga/designrulecheck/Netlist.java]: Fixed.

File: src/main/java/com/cburch/logisim/fpga/designrulecheck/Netlist.java (hunk 1 above)
Comment (MarcinOrlowski) [reply src/main/java/com/cburch/logisim/fpga/designrulecheck/Netlist.java]: Do agree, but not as part of this PR

**Review by BFH-ktt1 — APPROVED** [review]

**Pull request merged**